lib/winrm-fs/file_manager.rb

```diff
@@ -59,14 +59,14 @@ def delete(path)
       # Downloads the specified remote file to the specified local path
       # @param [String] The full path on the remote machine
       # @param [String] The full path to write the file to locally
-      def download(remote_path, local_path)
+      def download(remote_path, local_path, recurse = true)
```

Comment:
Is there a case where `download` is or should be called with `fal;se`?

Reply:
I can't think of a reason, i can remove that, I will add an integration test on Monday.

Reply:
cool and thanks for this work!